src/pages/academy/blog/index.js

```diff
@@ -57,13 +57,19 @@ const ArticlesPage = ({ data }) => {
         ? data.directus.blog.filter((item) => item.hide_for_eu == false)
         : data.directus.blog
 
+    const meta_attributes = {
+        og_title: 'Trading tips, guides, and more.',
+        og_description:
```

Comment:
(Question) Shouldn't we also localize these values?

Reply:
I think we shouldn't because the whole academy block with all articles has only English version...